bundles/org.openhab.binding.gpio/README.md

```diff
@@ -61,49 +150,130 @@ Note: If you are running Pigpio on same host as openHAB, then set host to **::1*
+                    instance has connectivity to OpenHAB.
 ```
 
-demo.sitemap:
+## Config file example
+
+Example for users who still prefer configuration files.
+
+demo.things:
 
-```perl
-sitemap demo label="Main Menu"
-{
-    Switch item=SampleInput1
-    Switch item=SampleOutput1
-}
 ```
+Thing gpio:pigpio-remote:mypi "MyPi GPIO" [ host="192.168.1.5", port=8888,
+                                                heartBeatInterval=10000,
+                                                inputConnectAction="REFRESH",      # REFRESH,NOTHING
+                                                inputDisconnectAction="NOTHING",   # SETUNDEF,NOTHING
+                                                inputReconnectAction="REFRESH",    # REFRESH,NOTHING
+                                                outputConnectAction="REFRESH",     # ALLOFF,ALLON,REFRESH,NOTHING
+                                                outputDisconnectAction="SETUNDEF", # SETUNDEF,NOTHING
+                                                outputReconnectAction="REFRESH" ]  # REFRESH,NOTHING
+    {
+        Channels:
+                Type pigpio-digital-output : BCM18 [ gpioId=18,invert=false,pulse=3000,pulseCommand="BLINK" ] # OFF,ON,BLINK
+
+                Type pigpio-digital-output : GPO4  [ gpioId=4, invert=true,pulse=5000,pulseCommand="OFF" ]
+                Type pigpio-digital-output : GPO17 [ gpioId=17,invert=false,pulse=500,pulseCommand="ON" ]
+                Type pigpio-digital-output : GPO27 [ gpioId=27,invert=false ]
+                Type pigpio-digital-output : GPO22 [ gpioId=22,invert=true ]
+
+                Type pigpio-digital-input  : GPI23 [ gpioId=23,debouncingTime=50,pullupdown="UP",invert=true ] # OFF,DOWN,UP
+                Type pigpio-digital-input  : GPI24 [ gpioId=24,debouncingTime=50,pullupdown="UP",invert=true ]
+                Type pigpio-digital-input  : GPI25 [ gpioId=25,debouncingTime=50,pullupdown="UP",invert=true ]
+                Type pigpio-digital-input  : GPI12 [ gpioId=12,debouncingTime=50,pullupdown="UP",invert=true ]
+                Type pigpio-digital-input  : GPI16 [ gpioId=16,debouncingTime=50,pullupdown="UP",invert=true ]
+                Type pigpio-digital-input  : GPI20 [ gpioId=20,debouncingTime=50,pullupdown="UP",invert=true,edgeMode="EDGE_EITHER ] # EITHER,RISING,FALLING
+                Type pigpio-digital-input  : GPI21 [ gpioId=21,debouncingTime=50,pullupdown="UP",invert=true,edgeMode="EDGE_RISING ]
+                Type pigpio-digital-input  : GPI5  [ gpioId=5, debouncingTime=50,pullupdown="UP",invert=true,edgeMode="EDGE_FALLING ]
+                Type pigpio-digital-input  : GPI6  [ gpioId=6, debouncingTime=50,pullupdown="UP",invert=true ]
+                Type pigpio-digital-input  : GPI13 [ gpioId=13,debouncingTime=50,pullupdown="DOWN",invert=false ]
+                Type pigpio-digital-input  : GPI26 [ gpioId=26,debouncingTime=50,pullupdown="OFF",invert=false ]
+    } 
+```
+
+demo.items:
+
+```
```

Comment:
```suggestion
```java
```

Reply:
Same. It seems this may have broken formatting?

Reply:
Same - the `java` keyword was suggested here, not at the end of the code fence.